turtlebot3_gazebo/package.xml

```diff
@@ -17,7 +17,7 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <depend>rclcpp</depend>
   <depend>gazebo_ros_pkgs</depend>
-  <depend>turtlebot3</depend>
+  <depend>turtlebot3_description</depend>
```

Comment:
As this is only a runtime dependency. Using `exec_depend` would be more appropriate